pkg/kubectl/cmd/rollingupdate/rollingupdate.go

```diff
@@ -25,7 +25,6 @@ import (
 	"k8s.io/klog"
 
 	"k8s.io/api/core/v1"
-	corev1 "k8s.io/api/core/v1"
```

Comment:
I have a slight preference for keeping corev1 instead-- "v1" is pretty ambiguous.

Reply:
Sure. Addressed. PTAL :)